### DISCUSSION TITLE
Does the v1.0 work on (old) Android devices?

### DISCUSSION THREAD

**Discussion by PinguThePenguin007** [nesbox/TIC-80#1905] [Q&A] [answered]
When i try to install the new TIC-80 version (from the Releases page) on my device, it throws a parse error, so i wanted to ask: are there any users expriencing the same problem? I do not want to create a new issue in case Nesbox has mentioned somewhere that TIC-80 is dropping the old versions support. Though, v0.9 ran perfectly for me.
I use Huawei y530-U00, it is powered by Android 4.3.

**Comment by nesbox** [comment, marked as answer]
We are using the SDL2 library for rendering TIC-80 on Android which has been updated to the latest version, this library requires Android 7.0 (min. SDK 24) as the minimum version. I don't know, we can try downgrading to Android 4.1 (minimum SDK 16), build it and see what happens :)

**Reply by PinguThePenguin007** [reply]
I thank you for your reply!
I'll definitely try that, though there is one thing that bothers me - what was the reason to update?
If TIC-80 now uses some of the new features the updated SDL2 library provides, this would make any attempts building it useless.
Thank you again for your reply!
